README.md

```diff
@@ -1 +1,65 @@
-# hello_microverse
\ No newline at end of file
+# hello_microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
The link to your project image is broken. Fixing this issue will make your project even more professional. You can upload an image to your repository and link to it. To learn more on this kindly see this [aricle](https://stackoverflow.com/questions/14494747/how-to-add-images-to-readme-md-on-github/53771381)